src/main/java/com/taf/core/ConfigManager.java

```diff
@@ -0,0 +1,54 @@
+package com.taf.core;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.taf.business.exceptions.ProcessingException;
+import com.taf.core.dto.ConfigDTO;
+import lombok.Getter;
+
+import java.io.FileInputStream;
+import java.util.Properties;
+
+public class ConfigManager {
+    private static final String CONFIG_PROPERTIES_FILE = "src/test/resources/config.properties";
+    private static ConfigManager instance = null;
+    @Getter
+    private ConfigDTO configDTO = null;
+
+    private ConfigManager() {
+        loadConfiguration();
+    }
+
+    public static ConfigManager getInstance() {
+        if (instance == null) {
+            instance = new ConfigManager();
+        }
+        return instance;
```

Comment:
why do you need `private static ConfigManager instance` variable at all? I'd suggest to initialize `private ConfigDTO configDTO` variable in this method instead and return instance of `ConfigDTO` (and rename method to `getConfigs` or something like that)

Reply:
Done